core/internal/gethwrappers/extract_contract_details.go

```diff
@@ -124,15 +124,10 @@ func replaceMetadata(beltArtifact []byte) (*binaryContractSections, error) {
 	if len(rv.String()) != len(rawBinary) {
 		panic("reconstruction with fixed metadata hash failed")
 	}
-	rawCode, err := hex.DecodeString(rv.staticData)
+	_, err := hex.DecodeString(rv.staticData)
 	if err != nil {
 		panic(errors.Wrapf(err, "can't fail, due to earlier hexutil.Decode check!"))
 	}
-	if !ascii.Match(rawCode) {
```

Comment:
@coventry this check was making some noise and I noticed you have the latest changes here. 
Do you have any insight into how important this check is and what the repercussions of removing it would be?

For some (possibly relevant) context, this PR includes the `create2` opcode in the Operator contract.

Reply:
Looks like it might be screwing with the go solidity wrappers as it is, looking at the failing tests.